privacy-configuration/tests.json

```diff
@@ -0,0 +1,630 @@
+{
+    "featuresDisabled": {
+        "name": "All features are disabled",
+        "desc": "Those tests use config1_reference.json where all features are disabled",
+        "referenceConfig": "config1_reference.json",
```

Comment:
this is new - I wanted to make it automatable to pair each test set with the right config